src/Umbraco.Core/Models/Rdbms/AuditEntryDto.cs

```diff
@@ -0,0 +1,53 @@
+﻿using System;
+using Umbraco.Core.Persistence;
+using Umbraco.Core.Persistence.DatabaseAnnotations;
+using Umbraco.Core.Persistence.DatabaseModelDefinitions;
+
+namespace Umbraco.Core.Models.Rdbms
+{
+    [TableName(TableName)]
+    [PrimaryKey("id")]
+    [ExplicitColumns]
+    internal class AuditEntryDto
+    {
+        public const string TableName = "umbracoAudit";
+
+        [Column("id")]
+        [PrimaryKeyColumn]
+        public int Id { get; set; }
+
+        [Column("performingUserId")]
+        public int PerformingUserId { get; set; }
+
+        [Column("performingDetails")]
+        [NullSetting(NullSetting = NullSettings.Null)]
+        [Length(1024)]
+        public string PerformingDetails { get; set; }
+
+        [Column("performingIp")]
+        [NullSetting(NullSetting = NullSettings.Null)]
+        [Length(64)]
+        public string PerformingIp { get; set; }
+
+        [Column("eventDate")]
+        [Constraint(Default = SystemMethods.CurrentDateTime)]
+        public DateTime EventDate { get; set; }
+
+        [Column("affectedUserId")]
+        public int AffectedUserId { get; set; }
+
+        [Column("affectedDetails")]
+        [NullSetting(NullSetting = NullSettings.Null)]
+        [Length(1024)]
+        public string AffectedDetails { get; set; }
+
+        [Column("eventType")]
+        [Length(256)]
+        public string EventType { get; set; }
```

Comment:
Probably should have an index on it so when we start querying on these things it is efficient ?

Reply:
general comment on indexes: haven't written queries yet and not sure how this is going to be used - was planning to add indexes when we add queries to the service/repo

Reply:
@zpqrtbnk The problem with that, is it means we'd have to wait until 7.10 to add these queries since we cannot make DB changes in patch releases. So IMO, we need to add indexes now so that we could potentially add these queries in a patch release.

... though, maybe adding queries and a report requires a minor version anyways?

Reply:
simple queries would be purely adding stuff so coud fit in a patch release, maybe... will thing about some kind of generic-enough indexes